src/main/resources/templates/search.solution

```diff
@@ -0,0 +1,35 @@
+//TODO SIDE-QUEST-5
+
+<w:table data="messages.getItems()">
```

Comment:
I would not add the whole block, but only the missing part like <th>User-Agent</th> and <td>                 @message.getUserAgent()</td> to keep it consistent with other solution files

Reply:
i would keep it like this because in the java files we have indicators where to place the code, but not here